pkg/config/validation/validation.go

```diff
@@ -556,41 +556,85 @@ func validateTLSOptions(tls *networking.ServerTLSSettings) (v Validation) {
 		if tls.CaCertificates != "" {
 			v = AppendValidation(v, fmt.Errorf("ISTIO_MUTUAL TLS cannot have associated CA bundle"))
 		}
+		if len(tls.TlsCertificates) > 0 {
+			v = AppendValidation(v, fmt.Errorf("ISTIO_MUTUAL TLS cannot have associated tlsCertificates"))
```

Comment:
We also need validation at Sidecar [resource](https://github.com/istio/istio/blob/master/pkg/config/validation/validation.go#L796) that multiple credential names/TlsCerts are not allowed